twill-core/src/main/java/org/apache/twill/internal/Configs.java

```diff
@@ -31,6 +31,11 @@
      */
     public static final String JAVA_RESERVED_MEMORY_MB = "twill.java.reserved.memory.mb";
 
+    /**
+     * Set this to false to disable the secure store updates done by default.
+     */
+    public static final String ENABLE_DEFAULT_SECURE_STORE_UPDATE = "twill.secure.store.update.location.enabled";
```

Comment:
It's better to have the constant name matches with the value. E.g. `SECURE_STORE_UPDATE_LOCATION_ENABLED`